### DISCUSSION TITLE
Gradle Build Location

### DISCUSSION THREAD

**Discussion by CtrlNate** [alfio-event/alf.io#1377] [Q&A] [answered]
I'm trying to build the app myself (using `./gradlew distribution`), and can't find the final build file I can use.

Is there some way to get the .war file that you folks include with releases?

**Comment by cbellone** [comment, marked as answer]
Hi!

the project layout follows [Gradle's standards](https://docs.gradle.org/current/userguide/gradle_directories.html), so the directory you're looking for is `./build/libs/`

However, if you're building the `main` branch, you won't find a `alfio-[version]-boot.war` file in the output directory. 

Starting from the next release, we will distribute alf.io as a JAR file and a Docker image.
The command to run the application does not change.

It would be great if you suggest modifications (with a Pull Request) to the README to clarify this point, so that it might be easier to other people to approach the project

Thank you!
Celestino
